sorc/orog_mask_tools.fd/orog_gsl.fd/module_gsl_oro_data_lg_scale.f90

```diff
@@ -611,9 +611,15 @@ subroutine calc_gsl_oro_data_lg_scale(tile_num,res_indx,halo)
       nu = 0
       nd = 0
       do jj = 1,jj_m
-         do ii = 1,ii_m/2   ! left half of box
-            if ( zs(ii,jj) > zs_mean ) nu = nu + 1
-         end do
+         if(mod(ii_m,2).eq.0.) then
+           do ii = 1,ii_m/2   ! left half of box
+              if ( zs(ii,jj) > zs_mean ) nu = nu + 1
+           end do
+         else
+           do ii = 1,ii_m/2+1   ! left half of box
```

Comment:
Just curious about your solution. It looks like you are including the column of points at (ii_m/2+1) in the both the left and right half of the box?

Reply:
Yes, that is correct.  And this occurs only when thera are an odd number of fine grid points within a coarse model grid cell.  The alternative would be not to count the center column at all in either left or right halves of the box.  The key is to have the same number of points counted in each half.